src/sinks/mod.rs

```diff
@@ -3,6 +3,8 @@ use snafu::Snafu;
 
 pub mod util;
 
+#[cfg(feature = "sinks-apex")]
+pub mod apex;
```

Comment:
Sorry, one more small change: can you add a corresponding variant to the big `Sinks` enum -- line 124 or so -- for this?

Reply:
Absolutely!